### PR TITLE
Removed gp_era details from gprecoverseg cli output

### PR DESCRIPTION
**Pull request by Annu149**
gprecoverseg prints master era details on stdout, removed era information from stdout and included it as debug message in gprecoverseg log file.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/bin/gppylib/gp_era.py [hunk 1]
```diff
@@ -145,7 +147,7 @@ def read_era(datadir, logger):
     if os.path.exists(erafile.filepath):
```
Comment (divyeshddv) [inline gpMgmt/bin/gppylib/gp_era.py:147]: Do we actually need to keep the era information? 
Would really like a comment explaining why we need the era info/file

If we really do need to keep era information, and the era file, then this change LGTM.

Comment (divyeshddv) [inline gpMgmt/bin/gppylib/gp_era.py:147]: Do we actually need to keep the era information? 
Would really like a comment explaining why we need the era info/file

If we really do need to keep era information, and the era file, then this change LGTM.

Comment (Annu149) [reply gpMgmt/bin/gppylib/gp_era.py:147]: There is another story already to explain why we need the era file.

**Comment by Annu149** [comment]
As discussed closing this PR

**Pull request closed without merging**